pyproject.toml

```
@@ -117,7 +117,7 @@ PyJWT = "~1.7"
 python-dateutil = "~2.8"
 tensorflow = "~2.3"
 tensorflow_hub = "~0.8"
-tensorflow-addons = "~0.10"
+tensorflow-addons = "~0.11"
```

`0.10` still works, doesn't it? It's just about the warning, isn't it?
```suggestion
tensorflow-addons = ">= 0.10, <= 0.12"
```